dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -250,6 +250,11 @@ DSRA4005.invalid.logintimeout=DSRA4005E: The server does not support the loginTi
 DSRA4005.invalid.logintimeout.explanation=The loginTimeout property is not supported when it is specified for a DataSource that is backed by a java.sql.Driver implementation because the server is unable to call getLoginTimeout on the java.sql.Driver class.
 DSRA4005.invalid.logintimeout.useraction=Modify the DataSource configuration to use a DataSource implementation instead of a java.sql.Driver implementation, or remove the loginTimeout property.
 
+# {0} the vendor class name, {1} name of the DataSource, 
+DSRA4006.null.connection=DSRA4006E: The java.sql.Driver {0} returned a null connection for {1} DataSource.
+DSRA4006.null.connection.explanation=Although the java.sql.Driver indicated it accepted the supplied URL, it returned a null connection.
+DSRA4006.null.connection.useraction=Ensure the JDBC Driver and DataSource configuration are correct and the expected java.sql.Driver class is being used.
```

Comment:
I don't understand how the user taking any of these actions will solve the problem.
Shouldn't it say something like, "This indicates a bug in the JDBC driver. Report the error to the JDBC driver vendor." ?

Reply:
By showing them their URL in the error message text (see my comment about that line), that will address the concern about wrong URL.  The specific error we are reporting here is a JDBC driver bug.  I don't think our useraction message should speculate on the possibility of a buggy JDBC driver accepting a bad URL and then later rejecting it.  It is best just to point out that the JDBC driver has a bug because that is what we are certain of.

Reply:
Given that it is appropriate for a JDBC driver to return null on a call to connect if it doesn't accept the URL format I think including that it returned true to acceptsURL is important